src/index.html

```diff
@@ -13,48 +13,44 @@
         class="card"
         data-qa="card"
       >
-        <div class="card__container">
-          <img
-            class="card__image"
-            src="./images/imac.jpeg"
-            alt="card"
-          >
-          <h2 class="card__title">
-             APPLE A1419 iMac 27" Retina<br>
-             5K Monoblock (MNED2UA/A)
-          </h2>
-          <p class="card__code">Product code: 195434</p>
-          <div class="card__stars-reviews">
-            <div class="stars">
-              <div class="stars__stars"></div>
-              <div class="stars__stars"></div>
-              <div class="stars__stars"></div>
-              <div class="stars__stars"></div>
-              <div class="stars__stars"></div>
-            </div>
-            <p class="card__reviews">
-              Reviews: 5
-            </p>
+        <img
+          class="card__image"
+           src="./images/imac.jpeg"
+           alt="card"
+         >
+        <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__stars-reviews">
+          <div class="stars">
+            <div class="stars__stars"></div>
+            <div class="stars__stars"></div>
+            <div class="stars__stars"></div>
+            <div class="stars__stars"></div>
+            <div class="stars__stars"></div>
           </div>
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
 
-          <div class="card__price">
-            <p class="card__price-title">
-               Price:
-            </p>
-            <p class="card__price-price">
-               $2,199
-            </p>
-          </div>
-          <div class="card__button">
-            <a
-              class="card__button-linc"
-              data-qa="hover"
-              href="#"
-              title="BUY">
-                BUY
-            </a>
-          </div>
+        <div class="card__price">
+          <p class="card__price-title">
+             Price:
+          </p>
+          <p class="card__price-price">
+             $2,199
+          </p>
         </div>
+        <a
+          class="card__button-link"
+          data-qa="hover"
+          href="#"
+          title="BUY">
+            BUY
```

Comment:
not fixed
<img width="822" alt="image" src="https://user-images.githubusercontent.com/53611910/205524209-68917003-7065-4f25-904a-48184b87e17d.png">